src/rred_reports/reports/tables.py

```diff
@@ -0,0 +1,208 @@
+from pathlib import Path
+
+import pandas as pd
+
+from rred_reports.masterfile import join_masterfile_dfs, parse_masterfile
+from rred_reports.reports.filler import TemplateFiller
+
+table_one_columns = [
+    "rred_user_id",
+    "pupil_no",
+    "entry_year",
+    "entry_gender",
+    "summer",
+    "entry_ethnicity",
+    "entry_language",
+    "entry_poverty",
+    "entry_special_cohort",
+    "exit_outcome",
+]
+
+table_two_columns = ["rred_user_id", "pupil_no", "entry_sen_status", "exit_outcome"]
+
+table_three_columns = ["rred_user_id", "pupil_no", "entry_date", "exit_date", "exit_num_weeks", "exit_num_lessons", "exit_outcome"]
+
+table_four_columns = [
+    "rred_user_id",
+    "pupil_no",
+    "exit_lessons_missed_ca",
+    "exit_lessons_missed_cu",
+    "exit_lessons_missed_ta",
+    "exit_lessons_missed_tu",
+    "total_lost_lessons",
+    "exit_outcome",
+]
+
+table_five_columns = [
+    "rred_user_id",
+    "pupil_no",
+    "entry_year",
+    "entry_bl_result",
+    "exit_bl_result",
+    "entry_li_result",
+    "exit_li_result",
+    "entry_cap_result",
+    "exit_cap_result",
+    "entry_wt_result",
+    "exit_wt_result",
+    "entry_wv_result",
+    "exit_wv_result",
+    "entry_hrsw_result",
+    "exit_hrsw_result",
+    "entry_bas_result",
+    "exit_bas_result",
+    "exit_outcome",
+]
+
+table_six_columns = [
+    "rred_user_id",
+    "pupil_no",
+    "exit_bl_result",
+    "month3_bl_result",
+    "month6_bl_result",
+    "exit_wv_result",
+    "month3_wv_result",
+    "month6_wv_result",
+    "exit_bas_result",
+    "month3_bas_result",
+    "month6_bas_result",
+    "exit_outcome",
+]
+
+
+def school_filter(whole_dataframe: pd.DataFrame, school_id):
+    """Function to filter school"""
+    return whole_dataframe[whole_dataframe.school_id == school_id]
+
+
+# specific filter for each table
+def filter_one(school_dataframe: pd.DataFrame):
+    """Filter for table one, for later use
+
+    Args: school_filter(pd.DataFrame)"""
+    return school_dataframe
+
+
+def filter_two(school_dataframe: pd.DataFrame):
+    """Filter for table two, for later use
+
+    Args: school_filter(pd.DataFrame)"""
+    return school_dataframe
+
+
+def filter_three(school_dataframe: pd.DataFrame):
+    """Filter for table three, for later use
+
+    Args: school_filter(pd.DataFrame)"""
+    return school_dataframe
+
+
+def filter_four(school_dataframe: pd.DataFrame):
+    """Filter for table four, for later use
+
+    Args: school_filter(pd.DataFrame)"""
+    return school_dataframe
+
+
+def filter_five(school_dataframe: pd.DataFrame):
+    """Filter for table five, for later use
+
+    Args: school_filter(pd.DataFrame)"""
+    return school_dataframe
+
+
+def filter_six(school_dataframe: pd.DataFrame):
+    """Filter for table six, for later use
+
+    Args: school_filter(pd.DataFrame)"""
+    return school_dataframe
+
+
+def summary_table_function(school_df: pd.DataFrame):

```

Comment:
I think maybe rename something like this?
```suggestion
def summary_table(school_df: pd.DataFrame):

```

Reply:
ah yes there is a 'summary_table' already in the code so the pytest didn't like this - will have a think though!

Reply:
ah fun `school_summary` maybe?